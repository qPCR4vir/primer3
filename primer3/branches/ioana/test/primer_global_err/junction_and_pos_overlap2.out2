../src/primer3_core: Cannot use both SEQUENCE_PRIMER_OVERLAP_POS and SEQUENCE_OVERLAP_JUNCTION_LIST constraints at the same time
