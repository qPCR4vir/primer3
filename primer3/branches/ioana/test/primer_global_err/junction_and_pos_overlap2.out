SEQUENCE_ID=Medicago Lyk3 mRNA INTRON_OVERLAP error small
SEQUENCE_TEMPLATE=cttgattcaatgaatctcaaaaatggattactattgttcattctgtttctggattgtgtttttttcaaagttgaatccaaatgtgtaaaagggtgtgatgtagctttagcttcctactatattataccatcaattcaactcagaaatatatcaaactttatgcaatcaaagattgttcttaccaattcctttgatgttataatgagctacaatagagacgtagtattcgataaatctggtcttatttcctatactagaatcaacgttccgttcccatgtgaatgtattggaggtgaatttctaggacatgtgtttgaatatacaacaaaagaaggagacgattatgatttaattgcaaatacttattacgcaagtttgacaactgttgagttattgaaaaagttcaacagctatgatccaaatcatatacctgttaaggctaagattaatgtcactgtaatttgttcatgtgggaatagccagatttcaaaagattatggcttgtttgttacctatccactcaggtctgatgatactcttgcgaaaattgcgaccaaagctggtcttgatgaagggttgatacaaaatttcaatcaagatgccaatttcagcataggaagtgggatagtgttcattccaggaagagatcaaaatggacatttttttcctttgtattctagaacaggtattgctaagggttcagctgttggtatagctatggcaggaatatttggacttctattatttgttatctatatatatgccaaatacttccaaaagaaggaagaagagaaaactaaacttccacaaacttctagggcattttcaactcaagatgcctcaggtagtgcagaatatgaaacttcaggatccagtgggcatgctactggtagtgctgccggccttacaggcattatggtggcaaagtcgacagagtttacgtatcaagaattagccaaggcgacaaataatttcagcttggataataaaattggtcaaggtggatttggagctgtctattatgcagaacttagaggcgagaaaacagcaattaagaagatggatgtacaagcatcgtccgaatttctctgtgagttgaaggtcttaacacatgttcatcacttgaatctggtgcggttgattggatattgcgttgaagggtcacttttcctcgtatatgaacatattgacaatggaaacttgggtcaatatttacatggtataggtacagaaccattaccatggtctagtagagtgcagattgctctagattcagccagaggcctagaatacattcatgaacacactgtgcctgtttatatccatcgcgacgtaaaatcagcaaatatattgatagacaaaaatttgcgtggaaaggttgctgattttggcttgaccaaacttattgaagttggaaactcgacacttcacactcgtcttgtgggaacatttggatacatgccaccagaatatgctcaatatggcgatgtttctccaaaaatagatgtatatgcttttggcgttgttctttatgaacttattactgcaaagaatgctgtcctgaagacaggtgaatctgttgcagaatcaaagggtcttgtacaattgtttgaagaagcacttcatcgaatggatcctttagaaggtcttcgaaaattggtggatcctaggcttaaagaaaactatcccattgattctgttctcaagatggctcaacttgggagagcatgtacgagagacaatccgctactacgcccaagcatgagatctatagttgttgctcttatgacactttcatcaccaactgaagattgtgatgatgactcttcatatgaaaatcaatctctcataaatctgttgtcaactagatgaagattttgtgtgacaaattgaattgtgtttgttaaaacatgtagaaagcatacaacaaatggtttgtactttacttgtatatgaaatattgcagttggagagtttttacttttcttacctcaattatccatcttgaacattgttttgtatgtggcaagagttcaaacactggtgtactcattgaaaagttatggtgagaaaatcactgatcagatgat
PRIMER_TASK=pick_detection_primers
PRIMER_PICK_RIGHT_PRIMER=1
PRIMER_PICK_INTERNAL_OLIGO=0
PRIMER_PICK_LEFT_PRIMER=1
PRIMER_FIRST_BASE_INDEX=1
SEQUENCE_OVERLAP_JUNCTION_LIST=10 15
SEQUENCE_PRIMER_OVERLAP_POS=10
PRIMER_ERROR=Cannot use both SEQUENCE_PRIMER_OVERLAP_POS and SEQUENCE_OVERLAP_JUNCTION_LIST constraints at the same time
=
